fortran/test/fetch_content_integration/test_micm_fort_api.F90

```diff
@@ -34,7 +35,7 @@ program test_micm_fort_api
 
 
   write(*,*) "[test micm fort api] Creating MICM solver..."
-  micm => micm_t(config_path, errcode)
+  micm => micm_t(config_path)
```

Comment:
There's no recourse here for when creating micm fails. The error function is called, I think, but the program still thinks that micm is valid.